### ISSUE TITLE
Unable to specify install path

### ISSUE THREAD

**Issue by bboles** [sttz/install-unity#28]
According to this code:

https://github.com/sttz/install-unity/blob/next/sttz.InstallUnity/Installer/Configuration.cs#L59

I should be able to specify `--opt installPathMac=<somepath>`.  I have been unable to get this to work and get this error:

```
/usr/local/bin/install-unity --yes install 2019.4.20f1 --packages Unity,iOS,Android --opt installPathMac='Unity.{major}.{minor}.{patch}'
Field value type System.String not yet supported.
```
I have tried various values for `installPathMac` but everything throws same error.  I also tried going the `config.json` route but could not figure out how to load `config.json`.

**Comment by sttz** [comment]
You're right, that's an oversight and setting `installPathMac` (and `downloadSubdirectory`) doesn't work properly.

You can call `install-unity --opt save` to create the config file with the default values at the right location and print the path to it. Then you should be able to edit `installPathMac` in the generated `config.json` file.

I'll fix this in an upcoming release.

**Comment by bboles** [comment]
Adjusting `installPathMac` in the `config.json` worked.  Thanks for the workaround!

Just a suggestion but you may want to make it more clear in your documentation on these points:
- The way to generate `config.json` is with the command `install-unity --opt save`.  It is a bit unclear because the `--help` text for the `--opt` option says to use `--opt <name>=<value>`.  Maybe consider using a different flag altogether for generating `config.json` and listing the available options since the syntax for these things are different.
- Clarify where the generated `config.json` is located.
- The `config.json` is loaded automatically if it exists, correct?  If so, maybe display a message at runtime that it is being used.  If I specify an option on the command-line, which takes precedence?

**Comment by sttz** [comment]
Thanks for the patience, this is now fixed in v2.9.0.

Also thanks for your suggestions. `--opt` is really intended as an advanced feature and I don't want to make it more prominent by adding multiple top-level options for it. For now, I expanded the examples in the help to make it more clear that the syntax is `--opt list` and `--opt save`.

There is already a message with the path to the config file when one is loaded but it's only shown when the verbosity is increased. The path to the config file is also shown when using `--opt save`. Options set on the command line take precedence over the value set in the config file.
